source/scss/320.scss

```diff
@@ -1,106 +1,44 @@
 header {
+
   @include transp-background();
   height: $height_header;
-  width: 100vw;
-  padding: 10px 20px;
-  display: flex;
   align-items: center;
-  justify-content: space-between;
 
-  .site-logo {
-    height: 100%;
-    z-index: -1;
+  .wrap {
 
-    img,
-    svg {
-      max-height: 100%;
-    }
-  }
-}
+    width: 100%;
 
-.menu {
-  &.main {
-    li {
-        a {
-            color: $primary_dark_50;
-            display: block;
-            font: $font_headings;
-            font-size: 1.5em;
-            padding: 10px 20px 12px;
-            text-decoration: none;
-
-            &:active,
-            &:hover,
-            &:focus {
-                background: $primary_dark_50;
-                color: #fff;
-            }
-      }
-    }
-  }
-}
+    .site-logo {
+      height: 80%;
 
-.nav-main {
-  .hamburger {
-    border: none;
-    height: 3.5em;
-    width: 3.5em;
-    padding: 0;
-    text-align: center;
-
-    &:active,
-    &:hover,
-    &:focus {
-      span {
-        color: #fff;
+      a {
```

Comment:
Add another hover effect, eg. a border or background colour, to make it really obvious that it's clickable

Reply:
fixed the hover issue so that it covers the whole logo! very obvious hover effect now!